std_msgs/Header header
int32 xian_dj_tele_op_left_server_cmd 
int32 xian_dj_tele_op_right_server_cmd 
int32 xian_dj_tele_op_up_server_cmd 
int32 xian_dj_tele_op_down_server_cmd
int32 xian_dj_tele_op_x_server_cmd 
int32 xian_dj_tele_op_b_server_cmd 
int32 xian_dj_tele_op_y_server_cmd 
int32 xian_dj_tele_op_a_server_cmd 
float32 xian_dj_tele_op_left_rocker_x_server_cmd 
float32 xian_dj_tele_op_left_rocker_y_server_cmd 
float32 xian_dj_tele_op_right_rocker_x_server_cmd 
float32 xian_dj_tele_op_right_rocker_y_server_cmd 
int32 xian_dj_tele_op_r1_server_cmd 
int32 xian_dj_tele_op_r2_server_cmd 
int32 xian_dj_tele_op_l1_server_cmd 
int32 xian_dj_tele_op_l2_server_cmd 
int32 xian_dj_tele_op_controller_client_tcp_heart_beat 